js/ui/toast/ui.toast.js

```diff
@@ -91,81 +90,55 @@ const Toast = Overlay.inherit({
     },
 
     _defaultOptionsRules: function() {
+        const tabletAndMobileAnimation = {
+            show: {
+                type: 'fade',
+                duration: 200,
+                from: 0,
+                to: 1
+            },
+            hide: {
+                type: 'fade',
+                duration: 200,
+                from: 1,
+                to: 0
+            }
+        };
+
         return this.callBase().concat([
             {
-                device: { platform: 'android' },
+                device(device) {
+                    return device.deviceType === 'phone';
+                },
                 options: {
+                    width: `calc(100vw - ${DEFAULT_MARGIN * 2}px)`,
```

Comment:
The `width` and `displayTime` properties have the same calculation in two places. Can we make them one time in the top of the function?

Reply:
I don't think we should take such small calculations anywhere. In addition, we use different widths (width and maximum width).

Reply:
You can add calculations in the extra variable and pass it to the width and maxWidth

Reply:
I suggest using relative values. For example:
`margin: 5%, width: 90%`
It will simplify the code and improve appearance on different devices

Reply:
We decided to change maxWidth on tables to 80%